src/logger.ts

```diff
@@ -13,9 +13,9 @@ const logger = createLogger({
     level: "info",
     format: combinedFormats,
     transports: [
-        new transports.File({ filename: "error.log", level: "error" }),
-        new transports.File({ filename: "info.log", level: "info" }),
-        new transports.File({ filename: "debug.log", level: "debug" })
+        new transports.File({ dirname: "logs", filename: "error.log", level: "error" }),
+        new transports.File({ dirname: "logs", filename: "info.log", level: "info" }),
+        new transports.File({ dirname: "logs", filename: "debug.log", level: "debug" })
```

Comment:
I am not sure if there is a benefit of having 3 different files, having all logs in a single file often makes debugging a lot easier

Reply:
Hmm, I take your point. The debug log does contain all the outputs though - we're logging more than we need to. I think it's quick nice to see just the errors, or just the info sometimes though.